### PR TITLE
Move the switch wrappers 1rem left

### LINKED ISSUE #11367: Add some margin between the Brave Payments section title and the switch
## Test plan

#11368 

----

### Description

Add some margin between the Brave Payments section title and the switch.

### PR DESCRIPTION
**Pull request by luixxiul**
This is required for `0.19.x` and `0.20.x` only.

screenshot 2017-10-09 1 00 30

Fixes #11367

Auditors:

Test Plan:

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Test Plan:

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by luixxiul** [comment]
To merger: please do not forget to merge this commit to `0.19.x` as well, thanks!

File: app/renderer/components/preferences/paymentsTab.js [hunk 1]
```diff
@@ -317,7 +317,11 @@ const styles = StyleSheet.create({
   },
 
   switchWrap: {
-    width: paymentStyles.width.tableCell
+    width: paymentStyles.width.tableCell,
+
+    // TODO: Remove this when #10913 is merged to master
+    position: 'relative',
+    left: '1rem'
```
Comment (cezaraugusto) [inline app/renderer/components/preferences/paymentsTab.js]: why is relative unit preferred?

File: app/renderer/components/preferences/paymentsTab.js [hunk 2]
```diff
@@ -365,8 +369,10 @@ const styles = StyleSheet.create({
   // History and settings icons
   switchWrap__mainIconsRight: {
     position: 'relative',
-    right: '12px',
-    top: '3.5px'
+    top: '3.5px',
```
Comment (cezaraugusto) [inline app/renderer/components/preferences/paymentsTab.js]: I know it was like this before does but this extra `.5px` makes any difference?

Comment (luixxiul) [reply app/renderer/components/preferences/paymentsTab.js]: I remember I used a screen ruler and adding it made it look nicer a bit.

File: app/renderer/components/preferences/paymentsTab.js (hunk 1 above)
Comment (luixxiul) [reply app/renderer/components/preferences/paymentsTab.js]: it's a magic number. It should work for now on 0.19 and 0.20.

Comment (cezaraugusto) [reply app/renderer/components/preferences/paymentsTab.js]: ok thanks

**Comment by cezaraugusto** [comment]
master n/a
0.20.x f0ce375017dc471d6503e15c38dce87bfdac091b
0.19.x 7d68d48f1ddfa31e9ca876a6136124a4826aec0b

**Pull request merged**